Palladium Rifts by Grinning Gecko/src/js/armor.js

```diff
@@ -32,6 +34,16 @@ async function applyDamageToNextActiveArmor(section, damage) {
   await setAttrsAsync(attrs);
 }
 
+on("change:repeating_armor:mvmt", async (e) => {
+  console.log("change:repeating_armor:mvmt", e);
+  const penalty = e.target.value;
+  const a = await getAttrsAsync(["repeating_movement_mph"]);
+  const newMvmt = a - (a * penalty);
```

Comment:
This won't work because `a` is an object. Also `a["repeating_movement_mph"]` will be a string, even though it's `input type="number"`. So you need to numberize it.

```suggestion
  const newMvmt = +a["repeating_movement_mph"] - (+a["repeating_movement_mph"] * penalty);
```